types/types.go

```diff
@@ -466,6 +466,18 @@ type ImageInspectInfo struct {
 	Os            string
 	Layers        []string
 	Env           []string
+	LayersDetail  []LayerDetail
+	History       []v1.History
+	Author        string
+	Size          int64
```

Comment:
Size of what? The compressed representation? Uncompressed representation? Either, depending on circumstances?

In OCI this can apparently be unknown.

Reply:
Still outstanding; the added comment doesn’t answer this question.